lambdas/user_access_patterns/lambda_for_createUser/index.js

```diff
@@ -1,6 +1,58 @@
-const AWS = require('aws-sdk');
-exports.handler = async (event) => {
-    // TODO implement
-    const response = (success, reason, data) => { return { success: success, reason: reason, data: data } }
-    return response(true, null, null);
+const AWS = require("aws-sdk");
+const crypto = require("crypto");
+//AWS.config.loadFromPath("../../../keys.json");
+//Use the below code for local setup
+AWS.config.update({
+  region: "local",
+  endpoint: "http://localhost:8000",
+});
+
+const dynamo = new AWS.DynamoDB.DocumentClient();
+function response(statusCode, message) {
+  return {
+    statusCode: statusCode,
+    body: JSON.stringify(message),
+  };
+}
+
+function hmac_hash(secretKey, password) {
+  const hashingSecret = secretKey;
+  const plainText = password;
+  const hashedStr = crypto
+    .createHmac("sha256", hashingSecret)
+    .update(plainText)
+    .digest("hex");
+
+  return hashedStr;
+}
+
+module.exports.createUser = (event, ctx, callback) => {
+  const reqBody = JSON.parse(event.body);
```

Comment:
since we are using an integration mapping template, this is not required.

Reply:
can you please tell me the replacement code for const reqBody that should be used, as per the integration template.

Reply:
event.body itself is an object so use it directly.

Reply:
`event.body` itself is a object hence you can use it directly

Reply:
Okay got it.